### ISSUE TITLE
Content review checklist 

### ISSUE THREAD

**Issue by evelinec** [OpenLiberty/draft-guide-rest-client-angular#3]
Content Review:

Review of the content and structure of the guide by a senior technical member to ensure the direction, flow and quality of the guide meet the requirements and expectations.

- [ ] Ensure guide's content coverage is adequate and suffice requirements
- [ ] Ensure code are structured well according to best practices
- [ ] Ensure direction of the guide is accurate
- [ ] Ensure flow of the guide is smooth
- [ ] Ensure requirements (purpose and goal) are clearly stated
- [ ] Ensure instructions make sense and are easy to understand
- [ ] Ensure concepts are being explained clearly first before describing what to learn
- [ ] Ensure the guide aligns with Open Liberty strategy
- [ ] Ensure the guide is consistent with other guides, especially the ones in the same category
- [ ] Ensure the instructions explain the why besides the what questions, identify areas that are not clear and hard to follow in first pass
- [ ] Ensure the guide in general follow and convey best practices and in line with accepted norms
- [ ] Ensure the title and abstract are well stated
- [ ] Ensure the guide has good Search Engine Optimization (SEO)

**Comment by gkwan-ibm** [comment]
#13

**Comment by evelinec** [comment]
Closed.
